### DISCUSSION TITLE
Backup process missing something?

### DISCUSSION THREAD

**Discussion by oren** [zeapo/Android-Password-Store#1248] [Q&A] [answered]
My laptop's hardware crashed and i am trying to restore my password store on a new laptop. My backup was created using the process [documented on the wiki](https://github.com/android-password-store/Android-Password-Store/wiki/Ubuntu-Android-Guide-backup). 

I restored the .password-store folder and the gpg keys using the commands on that wiki page:

Restore the pass data folder

```
cp -r ~/password-store ~/.password-store
```

Restore the gpg keys

```
gpg2 --import secret.gpg
gpg --edit-key <KEY_ID>
gpg>trust
enter '5'
```

but I see the following message:
```
gpg: decryption failed: No secret key
```

Here is the output of gpg -k:

```
gpg -k
/home/oren/.gnupg/pubring.kbx
-----------------------------
pub   rsa4096 2020-02-25 [SC]
      xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
uid           [ultimate] my name  <myname@gmail.com>
sub   rsa4096 2020-02-25 [E]
```

Any ideas what's going on? is the process of backup missing something? I am not familiar with managing gpg keys so any advice would be appreciated!

**Comment by msfjarvis** [comment]
How did you export your secret key? `gpg --export-secret-keys <key id>` is the only way to get our secret keys in a form that can be imported elsewhere.

**Reply by oren** [reply]
I believe I used the `gpg2 --export-secret-keys > secret.gpg` command. no key id.

**Reply by msfjarvis** [reply]
Not passing a key id should dump all your keys into a single file. I just did `gpg --export-keys > keys.gpg; gpg --import keys.gpg` and it correctly showed me that two secret keys were found but not imported because they exist already. Not really sure how that went wrong on your end.

**Comment by oren** [comment]
did you follow the wiki commands to restore them?
```
gpg2 --import keys.gpg
gpg --edit-key <KEY_ID>
gpg>trust
enter '5'
```

**Reply by msfjarvis** [reply]
Yes, that's what I wrote above. I did not need to trust the keys because they already existed in my computer, but the exported copy was valid.

**Comment by oren** [comment, marked as answer]
ok. thanks. at least the docs are accurate.

**Comment by oren** [comment]
I tried the import command again and this time i didn't leave the passphrase empty. now i see my private key and can access the password store. thank you!

**Reply by msfjarvis** [reply]
Glad to hear you were able to resolve it.
